pkg/southbound/defs.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package southbound
+
+import (
+	"github.com/openconfig/gnmi/client"
```

Comment:
is this needed ?

Reply:
Yes. For the target struct is needed. As you may noticed, I separated definitions from the actual code to make code more readable.

Reply:
+1